mixins/timeUtils.ts

```diff
@@ -92,6 +104,56 @@ const timeUtils = Vue.extend({
       } else {
         return `${minutes}:${formatDigits(seconds)}`;
       }
+    },
+    /**
+     * Returns the end time of an item
+     *
+     * @param {number} ticks - Ticks of the item to calculate
+     * @param {boolean} suffix - Whether to add or not the PM or AM prefix
+     * @returns {string} The resulting string
+     */
+    getEndsAtTime(ticks: number, suffix = true): string {
+      const ms = this.ticksToMs(ticks);
+      const endTimeLong = new Date(Date.now() + ms);
+      let format;
+      if (!suffix) {
+        format = endTimeLong.toLocaleString(this.$i18n.locale, {
+          hour: 'numeric',
+          minute: 'numeric'
+        });
+      } else {
+        format = this.$dateFns.format(Date.now() + ms, 'p');
+      }
+      // TODO: Use a Date object
+      return this.$t('endsAt', {
+        time: format
+      });
+    },
+    /**
+     * Returns the duration of an item in the following format: X hours Y minutes
+     *
+     * @param {number} ticks - Ticks of the item to calculate
+     * @returns {string} The resulting string
+     */
+    getRuntimeTime(ticks: number): string {
+      const ms = this.ticksToMs(ticks);
+      return this.$dateFns.formatDuration(
+        intervalToDuration({ start: 0, end: ms }),
+        {
+          format: [this.$t('hours'), this.$t('minutes')]
```

Comment:
The documentation for date-fns doesn't suggest these should be translated. They seem to be simple references to Duration object properties.

Reply:
Those strings were there before my PR, just untranslated. Can't find the documentation that suggests how to use this method, can you point it to me please?

Reply:
Here it is: https://date-fns.org/v2.16.1/docs/formatDuration

They're not strings that are shown to the user, they simply specify what parts of the duration to show. The locale parameter is what's used to define the language.